client/src/Components/UserProfile/Checkbox.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Checkboxx = ({ type = 'checkbox', checked = false, onChange, id }) => {
```

Comment:
Any reason why there is two `x`?

Reply:
oh i see - naming conflict with Material UI. you can name it something else like `AppCheckbox` or `AvailabilityCheckbox` something a bit more understandable

Reply:
in addition - you can import these named constants differently like
`import Checkbox as AvailabilityCheckbox from "./Checkbox.js"`